dev/com.ibm.websphere.appserver.features/visibility/public/opentracing-1.3/com.ibm.websphere.appserver.opentracing-1.3.feature

```diff
@@ -19,5 +19,5 @@ IBM-API-Package: io.opentracing;  type="third-party",\
          com.ibm.websphere.appserver.thirdparty.opentracing.0.31.0; location:="dev/api/third-party/,lib/"; mavenCoordinates="io.opentracing:opentracing-api:0.31.0"
 -jars=com.ibm.websphere.appserver.spi.opentracing.1.3; location:=dev/spi/ibm/
 -files= dev/spi/ibm/javadoc/com.ibm.websphere.appserver.spi.opentracing.1.3_1.0-javadoc.zip
-kind=noship
+kind=ga
```

Comment:
opentracing-1.3 isn't part of MP 2.2

Reply:
mpOpenTracing-1.3 is depending on opentracing-1.3.